tools/wptrunner/wptrunner/executors/base.py

```diff
@@ -80,9 +80,10 @@ def __call__(self, test, result, extra=None):
 testharness_result_converter = TestharnessResultConverter()
 
 
-def hash_screenshot(data):
+def hash_screenshots(screenshots):
     """Computes the sha1 checksum of a base64-encoded screenshot."""
-    return hashlib.sha1(base64.b64decode(data)).hexdigest()
+    return [hashlib.sha1(base64.b64decode(screenshot)).hexdigest()
+            for screenshot in screenshots]
 
 
 def _ensure_hash_in_reftest_screenshots(extra):
```

Comment:
I haven't reviewed the rest of this file as it seems WIP - let me know if there's any part that needs looked at.